src/config/dynamic-theme-fixes.config

```diff
@@ -3902,6 +3902,38 @@ INVERT
 
 ================================
 
+jumia.com
+jumia.co
+jumia.ma
+jumia.ci
+jumia.dz
+jumia.ug
+jumia.sn
+zando.co.za
+
+INVERT
+svg.ic[role="img"]:not([aria-label="Zando logo"])
+.logo > a > img
```

Comment:
Logo at cart page (while signed-in)
![Cart logo](https://user-images.githubusercontent.com/14346945/100476457-4dc46b80-30ee-11eb-82f5-1ccc7b4b934c.jpg)
![Cart logo (2)](https://user-images.githubusercontent.com/14346945/100476460-4ef59880-30ee-11eb-8cc2-77b01b93b472.jpg)